### ISSUE TITLE
[COGNITION]: GIBCT® - Consider making the tuition and housing breakdowns into headings and semantic lists

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1277]
## Description

Our tuition and housing cost lists are marked up with divs and headings right now. I think it would be beneficial to make them into ~~definition lists~~ headings and heading-labeled lists for screen reader context. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a tester, I want there to be zero aXe errors when scanned with browser plugin or CI/CD tests.
* As a screen reader user, I want to hear ~~the definition lists~~ headings and lists properly announced

## Possible Markup Change
I think it might be worth looking at this layout, switching the **Total paid to you** to an H4 + P like the other headings in the Your estimated benefits section, and using lists on the Estimated benefits per term section. 

/cc @CrystabelReiter for visibility

```html
<h4 id="tuition-and-fees">Tuition and fees</h4>
<a href="#">Learn more</a>
<ul aria-labelledby="tuition-and-fees" role="list">
  <li>
    <span>Fall:</span>
    <span>$3,548</span>
  </li>
  <li>
    <span>Spring:</span>
    <span>$3,548</span>
  </li>
  <li>
    <span>
      <b>Total per year:</b>
    </span>
    <span>$7,096</span>
  </li>
</ul>
```

## Screenshots or Trace Logs

![Screen Shot 2020-09-08 at 3.34.20 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/167f9111-46e3-4d8b-8dcc-4685074616e8)

**Comment by 1Copenut** [comment]
@ksantiagomorgan6 && @desireeturner 

I think it might be worth looking at this layout, switching the **Total paid to you** to an H4 + P like the other headings in the Your estimated benefits section, and using lists on the Estimated benefits per term section. 

```html
<h4 id="tuition-and-fees">Tuition and fees</h4>
<a href="#">Learn more</a>
<ul aria-labelledby="tuition-and-fees" role="list">
  <li>
    <span>Fall:</span>
    <span>$3,548</span>
  </li>
  <li>
    <span>Spring:</span>
    <span>$3,548</span>
  </li>
  <li>
    <span>
      <b>Total per year:</b>
    </span>
    <span>$7,096</span>
  </li>
</ul>
```

---
![Screen Shot 2020-09-08 at 3.34.20 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/167f9111-46e3-4d8b-8dcc-4685074616e8)

**Comment by desireeturner** [comment]
@1Copenut Would you mind either entering a new issue to cover the changes you've requested in https://github.com/department-of-veterans-affairs/va.gov-team/issues/1277#issuecomment-689122144 or updating this issue to reflect the latest just so we're clear on the intent of what we're wanting to accomplish given the latest version of EYB? Thank you!

**Comment by 1Copenut** [comment]
@desireeturner I updated the ticket title and description so we don't have to close and track a new issue.

**Comment by sonntag-adam** [comment]
![Screen Shot 2020-09-10 at 11.22.01 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/7aafc07e-f2ca-41e4-a898-ad5472101d39)
##
![Screen Shot 2020-09-10 at 11.22.01 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/d5647499-54fa-4cc2-9f05-b328ee8c7161)

**Comment by sonntag-adam** [comment]
@1Copenut this is now in staging.

**Comment by 1Copenut** [comment]
@sonntag-adam The H4 + P looks 💯 in staging. Was there any discussion about refactoring the Tuition & Fees, Housing, and Book Stipend divs into unordered lists? It's not a big priority, but would help make a stronger association with the headings if the code was refactored to match the HTML snippet in the description.

LMK if that's icebox work, or you'd like another ticket.

**Comment by ksantiagomorgan6** [comment]
Can we get another ticket for that?

**Comment by 1Copenut** [comment]
@ksantiagomorgan6 https://github.com/department-of-veterans-affairs/va.gov-team/issues/13761

**Comment by ksantiagomorgan6** [comment]
Thanks @1Copenut

**Comment by desireeturner** [comment]
@1Copenut Can this issue be closed now that #13761 is open?

**Comment by 1Copenut** [comment]
@desireeturner Yes, this one can be closed. I'll take care of that now.
